_episodes/03-packaging-installing.md

```diff
@@ -313,7 +317,27 @@ print(temperature.fahr_to_celsius(100))
 ~~~
 {: .language-python}
 
-Now we can import from within this folder, but only if our working directory is at the top level `conversions` 
+Even if the `__init__.py` file is empty, its existence indicates to Python that we can import names
+from that package. However, by adding import code to it, we can make our package easier to use.
+Add the following code to the init file:
+
+~~~
```

Comment:
````suggestion
```
````

a previous contributor used the `~`,  but i think it will remain more compatible if we swithc them to backticks more consistently

Reply:
Noted, I ran a find and replace on the file (a393a08) and will keep that in mind for future PRs.